src/test/kotlin/no/nav/familie/ef/sak/iverksett/IverksettingDtoMapperTest.kt

```diff
@@ -107,11 +137,270 @@ internal class IverksettingDtoMapperTest {
+
+
+        assertThat(vedtak.tilkjentYtelse?.startdato).isEqualTo(LocalDate.of(2022, 4, 7))
+        //opphørsdato ikke i bruk?
+
+        assertThat(vedtak.tilkjentYtelse?.andelerTilkjentYtelse?.size).isEqualTo(1)
+
+        assertThat(vedtak.vedtaksperioder.size).isEqualTo(1)
+        val vedtaksperiode = vedtak.vedtaksperioder.first()
+        assertThat(vedtaksperiode.fraOgMed).isEqualTo(LocalDate.of(2022, 3, 27))
+        assertThat(vedtaksperiode.tilOgMed).isEqualTo(LocalDate.of(2022, 3, 28))
+        assertThat(vedtaksperiode.aktivitet.name).isEqualTo(AktivitetType.BARN_UNDER_ETT_ÅR.name)
+        assertThat(vedtaksperiode.periodeType.name).isEqualTo(VedtaksperiodeType.HOVEDPERIODE.name)
+        //assertThat(iverksettDto.vedtak.vedtakstidspunkt) - sjekker ikke denne da det er LocalDate.now()
+    }
+
+    private fun mockReturnerObjekterMedAlleFelterFylt(): UUID? {
+        val grunnlagsdata =
+                opprettGrunnlagsdata().copy(søker = søker(), barn = listOf(barnMedIdent(fnr = "123", navn = "fornavn etternavn")))
+        every { grunnlagsdataService.hentGrunnlagsdata(any()) } returns GrunnlagsdataMedMetadata(grunnlagsdata,
+                                                                                                 false,
+                                                                                                 LocalDateTime.parse("2022-03-25T05:51:31.439"))
+        every { arbeidsfordelingService.hentNavEnhetIdEllerBrukMaskinellEnhetHvisNull(any()) } returns "4489"
+        val behandlingId = UUID.fromString("73144d90-d238-41d2-833b-fc719dae23cb")
+
+        val behandlingBarn = objectMapper.readValue<BehandlingBarn>(behandlingBarnJson)
+        every { barnService.finnBarnPåBehandling(any()) } returns listOf(behandlingBarn)
+        every { barnMatcher.kobleBehandlingBarnOgRegisterBarn(any(), any()) } returns listOf(
+                MatchetBehandlingBarn(
+                        fødselsnummer = "1234",
+                        barn = barnMedIdent(fnr = "1234", "fornavn etternavn"),
+                        behandlingBarn = behandlingBarn
+                )
+        )
+        every { vilkårsvurderingRepository.findByBehandlingId(any()) } returns listOf(objectMapper.readValue(vilkårsvurderingJson))
+        every { vedtakService.hentVedtak(any()) } returns objectMapper.readValue(vedtakJson)
+        every { brevmottakereRepository.findByIdOrNull(any()) } returns objectMapper.readValue(brevmottakereJson)
+        every { tilbakekrevingService.hentTilbakekreving(any()) } returns objectMapper.readValue(tilbakekrevingJson)
+        every { simuleringService.hentLagretSimuleringsoppsummering(any()) } returns objectMapper.readValue(
+                simuleringsoppsummeringJson)
+        every { tilkjentYtelseService.hentForBehandling(any()) } returns objectMapper.readValue(tilkjentYtelseJson)
+        return behandlingId
+    }
+
+
+    private val saksbehandlingJson = """
```

Comment:
Burde vi legge disse i egne json-filer?

Reply:
Egne json-filer: Smak og behag. Men jeg liker å ha det i samme fil når det er kun den testen det gjelder.
Kan new-e opp alle objektene inkludert saksbehandling, men jeg synes det blir mer lesbart å liste opp alle felter og verdier med json i stedet, spesielt for objekter med mange felter (som igjen har lister med objekter med flere felter). Det er sikkert smak og behag det også, så jeg er åpen for å endre det også :)